src/core/qgsproject.cpp

```diff
@@ -1951,7 +1986,10 @@ bool QgsProject::writeProjectFile( const QString &filename )
   QDomElement qgisNode = doc->createElement( QStringLiteral( "qgis" ) );
   qgisNode.setAttribute( QStringLiteral( "projectname" ), title() );
   qgisNode.setAttribute( QStringLiteral( "version" ), QStringLiteral( "%1" ).arg( Qgis::version() ) );
-
+  QString newSaveUser = QgsApplication::userLoginName();
+  QString newSaveUserFull = QgsApplication::userFullName();
+  qgisNode.setAttribute(QStringLiteral("save-user"), newSaveUser);

```

Comment:
Just a question: is that dashed style consistent with the majority of tag/attrs name in the project XML? I see there are many camel-case items too.

Reply:
There is a mix of different styles. Some have the dashed style, some camelcase and some are just written in lowercase like <customproperties>. Are perhaps the tags with the dashed style like <layer-tree-group>,<layer-tree-layer>,<snapping-settings> older ones and the camelcase items like <ProjectViewSettings>,<projectCrs>,<homePath> mainly newer ones?